src/lib/mathlib/math/filter/MedianFilter.hpp

```diff
@@ -59,8 +59,8 @@ class MedianFilter
 
 	void insert(const T &sample)
 	{
-		_head = (_head + 1) % WINDOW;
 		_buffer[_head] = sample;
+		_head = (_head + 1) % WINDOW;
```

Comment:
Why?

(it doesn't actually matter in this case, I'm just curious)

Reply:
No, why is it opposite in the first place? Is there some trickery? I was just confused for a moment while debugging unit tests because the array didn't contain what I expected.